cypress/integration/login_activity_spec.js

```diff
@@ -92,4 +92,42 @@ describe("LoginActivity spec", () => {
 
     cy.get("div.dashboard");
   });
+  it("should display the logout confirmation when the logout icon is clicked", () => {
+    //If the reduce menu icon is present, open the menu page and click on signout icon
+    // else click directly on the signout icon 
+    if (cy.get("[class=appHeader__identified__trigger]")) {
+      cy.get("[class=appHeader__identified__trigger]").click();
+      cy.get("[id=signout_icon]").click();
+    } else {
+      cy.get("[id=signout_icon]").click();
+    }
+    //check if the confirmation dialog is rendered
+    cy.get(".MuiDialog-paper");
+    //dismiss the confirmation dialog
+    cy.get(".reset_button button").click();
```

Comment:
I think that should be more accurate have .MuiDialog-paper .reset_button button in order to exclude the others .reset_button button not in the scope.

Reply:
OK

Reply:
Ok, I change it. thanks